microlith/loki/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -273,3 +273,19 @@ groups:
           title: "Large Number of Dropped Ticks Detected (node: {{ $labels.hostname }})"
           description: Couchbase Server nodes regularly send heartbeat ticks to each other. Dropped ticks are usually a sign of resource contention.
           remediation: Increase number of CPUs available to Couchbase Server or, if you are running a virtualised environment, check for VM overcommitment.
+
+      - alert: CB90067-serviceRollback
+        expr: |
+          count_over_time( {file="memcached.log"} |= "requires rollback to seqno:0" != "eq_dcpq:replication" [24h]) > 0
```

Comment:
If the rule works over `[24h]`, that'll mean that every time Loki evaluates it (which I believe is every minute by default) it'll need to scan all the memcached logs from the past 24 hours. We might want to drop this down to 1h or similar - even if the alert is resolved, whatever's on the receiving end (email / Slack / PagerDuty / Jira / whatever) will still have a record of it.

Reply:
I will set it to an hour :)